napari/_qt/qt_main_window.py

```diff
@@ -53,6 +53,7 @@
 from napari._qt._qapp_model import build_qmodel_menu
 from napari._qt._qapp_model.qactions import init_qactions
 from napari._qt.dialogs.confirm_close_dialog import ConfirmCloseDialog
+from napari._qt.dialogs.preferences_dialog import PreferencesDialog
```

Comment:
If we're only using this for the return type annotation we should put it under `if TYPE_CHECKING`

Reply:
Ah this is a pre-commit change. `PreferencesDialog` is actually used but the import is inside the function. Linting did not like my new top of file import to be under type checking as it is used for more than type hinting.

Having the import outside does not seem to create circular import but if we want to be safe I can move the import under type checking and add a lint ignore?

Reply:
Pre-commit do this because you add it as a return type annotation.

Reply:
Since it's actually being used below I think it's fine to just leave it at module level. I didn't get any circular imports or anything and tests are passing so I'm approving now.